service/collector/collector.go

```diff
@@ -34,30 +40,48 @@ var (
 )
 
 type Config struct {
-	Logger  micrologger.Logger
-	Watcher func(opts metav1.ListOptions) (watch.Interface, error)
+	K8sClient kubernetes.Interface
+	Logger    micrologger.Logger
+	Watcher   func(opts metav1.ListOptions) (watch.Interface, error)
+
+	// EnvironmentName is the name of the Azure environment used to compute the
+	// azure.Environment type. See also
+	// https://godoc.org/github.com/Azure/go-autorest/autorest/azure#Environment.
+	EnvironmentName string
```

Comment:
Does it make sense to change the name in a single place? We called it `Cloud` and I think having it different in a single place doesn't make any good even though it's closer to original API and I like it. If it was changed everywhere I would 100% agree. I had to jump back and forth do discover what it is.

Reply:
I would like to start using the better name.

Reply:
> I would like to start using the better name.

Can you please then add TODO comments into place, where we call it cloud?

Reply:
Good shout. Here you go: https://github.com/giantswarm/giantswarm/issues/4124. Added a couple of comments in the code as well.